core/src/scene/light.h

```diff
@@ -27,6 +27,15 @@ enum class LightOrigin {
     world
 };
 
+static std::string lightOriginString(LightOrigin origin) {
```

Comment:
Alas, I really wish c++ typed enums allowed this - Java enums do and it's very very convenient.